client/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:14
+
+WORKDIR /app
+
+COPY package*.json ./
```

Comment:
what is the reason for multiple copies?
Why dont you utilize multistage here also?